scripts/release/preparepackages.js

```diff
@@ -193,6 +194,24 @@ const tasks = new Listr( [
 							concurrency: cliArguments.concurrency
 						} );
 					}
+				},
+				{
+					title: 'Preparing CDN files.',
+					task: async () => {
+						await fs.copy( './dist/browser', './release_cdn/' );
+						await fs.copy( './dist/translations', './release_cdn/translations/' );
+						await fs.copy( './dist/browser', './release_zip/ckeditor5/' );
+						await fs.copy( './dist/translations', './release_zip/ckeditor5/translations/' );
+						await fs.copy( './scripts/assets/zip', './release_zip/' );
+
+						await fs.ensureDir( './release_cdn/zip' );
+						await tools.shExec(
+							`cd release_zip && zip -r ../release_cdn/zip/ckeditor5-${ latestVersion }.zip ./*`,
+							{ verbosity: 'error' }
+						);
+
+						await fs.copy( './build', './release_cdn/dll/ckeditor5-dll/' );
```

Comment:
It would be good to move some paths to `scripts/release/utils/constants.js`. At least `release_cdn` and `release_zip` names occur in several places.